intake/catalog/utils.py

```diff
@@ -133,6 +174,39 @@ def expand_defaults(default, client=False, getenv=True, getshell=True):
     return default
 
 
+def merge_pars(params, user_inputs, spec_pars, client=False, getenv=True,
+               getshell=True):
+    context = params.copy()
+    for par in spec_pars:
+        val = user_inputs.get(par.name, par.default)
+        if val is not None:
+            if isinstance(val, six.string_types):
```

Comment:
are we moving towards supporting python2?

Reply:
It would be nice to set a direction on this soon. I have been using `six` in my PRs also, since I see it used in various places in the codebase, but not bothering to actually test it since there is no CI for Python 2. https://github.com/intake/intake/issues/228

Reply:
Right. I have been using Python 3 features when they are convenient to me - but I'll stop if we plan to ever support Python 2 :)

Reply:
OK, final: no py2. The six things can migrate away, although they do no harm.